src/main/java/core/basesyntax/service/impl/DataValidatorServiceImpl.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.service.impl;
+
+public class DataValidatorServiceImpl {
```

Comment:
```suggestion
public class DataValidatorImpl implements DataValidator {
```
let's create an interface as well